src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
-
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
```

Comment:
Keys in Enum should start with a capital letter
```suggestion
enum Shape {
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle',
```